packages/sysutils/busybox/scripts/init

```diff
@@ -248,8 +248,8 @@ mount_part() {
 
 mount_sysroot() {
   if [ "$SYSTEM_TORAM" = "yes" ]; then
-    cp /flash/$IMAGE_SYSTEM /dev/$IMAGE_SYSTEM
-    mount_part "/dev/$IMAGE_SYSTEM" "/sysroot" "ro,loop"
+    cp /flash/$IMAGE_SYSTEM /dev/${SYSTEM_IMAGE//\//_}
+    mount_part "/dev/${SYSTEM_IMAGE//\//_}" "/sysroot" "ro,loop"
   else
```

Comment:
Huh, this looks a lot like it will break default toram behaviour - SYSTEM_IMAGE won't be set unless you explicitly add it to cmdline.

Why not just use /dev/SYSTEM?

Reply:
Try to imaging what will happen if you name your image eg `null` and pass that in via SYSTEM_IMAGE.

Hint: just using `/dev/SYSTEM` will fix that too and it's clear which file under dev will be used by LE